frappe/email/receive.py

```diff
@@ -425,7 +425,7 @@ def __init__(self, content):
 		self.set_content_and_type()
 		self.set_subject()
 		self.set_from()
-		self.message_id = (self.mail.get("Message-ID") or "").strip(" <>")
+		self.message_id = (self.mail.get("Message-ID") or "").strip("\n\r\t <>")
```

Comment:
Why not just strip all whitespace characters? (I'm not sure if that'd work) but I'd trust to use `string.whitespace` over adding more characters here over time 😅.